lib/screens/profile_screen.dart

```diff
@@ -17,36 +18,29 @@ class _ProfileScreenState extends State<ProfileScreen> {
   List questionsList = [];
   List answersList = [];
   List categories = [];
+  String firstName = ""; // this will be changed later
+  bool loading = true;
 
   @override
   void initState() {
-    initGetQuestions();
-    initGetAnswers();
+    initGetProfileInfo();
     initGetActiveCategories();
     super.initState();
   }
 
-  void initGetQuestions() {
-    ProfileService().getProfileQuestions().then(
+  void initGetProfileInfo() {
+    ProfileService().getProfileInformation().then(
       (payload) {
         setState(() {
-          questionsList = payload;
+          questionsList = payload[0];
+          answersList = payload[1];
+          firstName = payload[2];
```

Comment:
instead of using just the firstName, can you use the entire Users List to populate these as well? 
![image](https://user-images.githubusercontent.com/36316432/112362614-dc5a8b00-8c91-11eb-91f2-9eb39c1def3d.png). The name, email, profile_url, and rating should be from the View Model. 

And also use the List<dynamic> category to populate interests. You can copy the users_model from my branch.